OracleWebLogic/samples/12213-domain-home-in-image/Dockerfile

```diff
@@ -53,8 +57,7 @@ RUN chmod +xw /u01/oracle/*.sh && \
     chown -R oracle:oracle $DOMAIN_HOME && \
     chmod -R a+xwr $DOMAIN_HOME
 
-COPY properties/domain.properties ${PROPERTIES_FILE_DIR}/
-COPY properties/domain_security.properties ${PROPERTIES_FILE_DIR}/
+COPY properties/docker_build/domain*.properties ${PROPERTIES_FILE_DIR}/
```

Comment:
No for security reasons those are meant to pass in only at runtime and not when the image is built.

Reply:
The credentials property file are copied into the image during build and then deleted so that the credentials are not present in the image after it is built.